api/src/main/java/org/pmiops/workbench/cdr/model/DbCriteria.java

```diff
@@ -119,7 +121,39 @@ public void setCount(String count) {
 
   @Transient
   public Long getLongCount() {
-    return Strings.isNullOrEmpty(this.count) ? null : Long.valueOf(this.count);
+    return getLongValue(this.count);
```

Comment:
If we change those we would no longer need these Transient properties for count, rollup count and item count.